packages/clerk-js/src/core/clerk.ts

```diff
@@ -1159,6 +1220,25 @@ export default class Clerk implements ClerkInterface {
     return this.setActive({ session: null });
   };
 
+  public authenticateWithGoogleOneTap = async (
+    params: AuthenticateWithGoogleOneTapParams,
+  ): Promise<SignInResource | SignUpResource> => {
+    return this.client?.signIn
+      .create({
+        strategy: 'google_one_tap',
+        token: params.token,
+      })
+      .catch(err => {
+        if (isClerkAPIResponseError(err) && err.errors[0].code === 'external_account_not_found') {
+          return this.client?.signUp.create({
```

Comment:
@panteliselef I think I've mentioned this before, but I'm double-checking that this `signUp.create` will trigger a Captcha functionality, and if the user has selected the "Smart" option it will try to find the `clerk-captcha` in the DOM. Thankfully, it won't break, because we fallback to "Invisible", but the fallback mechanism may change in the future, we should not depend on it.

Reply:
ACK